packages/material-ui-codemod/src/v5.0.0/box-sx-prop.js

```diff
@@ -0,0 +1,114 @@
+const boxProps = [
```

Comment:
https://github.com/mui-org/material-ui/pull/23451 will add `typography`

Reply:
That won't affect migrating from 4.x though.

Reply:
Ah correct :)